05LessonFive/Checkpoint1/index.html

```diff
@@ -9,14 +9,18 @@
 <body>
     <table>
         <tr>
+            <!--picture of berries-->
             <td width="500px">
                 <img src="img/berries.jpg" width="500px" />
             </td>
+            <!--box with text and link-->
             <td class="grey_box" width="500px">
                 <h1 align="center">Berries</h1>
                 <p align="center">
                     The snozzberries taste like snozzberries! You should totally read this article on Cracked.com! Childhood ruined. You're welcome.
                     <center>
+
+                        <!--read more button-->
```

Comment:
Great job commenting! any reason you went with tables? also, thanks for ENHANCING my childhood. the fact that snozzberries are dicks is pretty great.